napari/_qt/qt_viewer.py

```diff
@@ -111,6 +106,31 @@ def __init__(self, viewer):
 
         self.setAcceptDrops(True)
 
+    def set_camera(self):
+        print(self.viewer.dims.display)
+        print(sum(self.viewer.dims.display))
+        if sum(self.viewer.dims.display) > 2:
```

Comment:
Is this working as expected? It should be correct.

For future proofing, can you made the statement
```python
if ... == 2:
   # Set a 2D camera
   ....
elif ... == 3:
   # Set a 3D camera
   ....
else:
   raise an error
```

Reply:
changed